src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+interface Props {
+  perPage: number;
+  total: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  perPage,
+  total,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesList: number[] = getNumbers(1, Math.ceil(total / perPage));
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={
+            currentPage === 1 ? undefined : () => onPageChange(currentPage - 1)
+          }
```

Comment:
Using 'href="#prev"' may lead to unwanted behavior as it modifies the URL. It would be better to use a button element instead of an anchor tag for the pagination controls if they do not navigate to a different URL.

Reply:
not valid for task